packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -412,8 +415,16 @@ class RenderParagraph extends RenderBox
     RenderBox child = firstChild;
     while (child != null) {
       final TextParentData textParentData = child.parentData;
-      final Matrix4 transform = Matrix4.translationValues(textParentData.offset.dx, textParentData.offset.dy, 0.0)
-        ..scale(textParentData.scale, textParentData.scale, textParentData.scale);
+      final Matrix4 transform = Matrix4.translationValues(
+        textParentData.offset.dx,
+        textParentData.offset.dy,
+        0.0,
+      )
+        ..scale(
```

Comment:
I am not sure if how we indent in this situation, do we indent in this case?

Reply:
Fixed.